.changeset/famous-masks-mate.md

```diff
@@ -0,0 +1,18 @@
+---
+"emails-and-messages": major
+"data-importer": major
+"products-feed": major
+"@saleor/apps-shared": major
+"invoices": major
+"klaviyo": major
+"segment": major
+"app-avatax": major
+"cms-v2": major
+"search": major
+"app-taxjar": major
+"slack": major
+"taxes": major
+"crm": major
+---
+
+Prepare each app to handle user without "MANAGE_APPS" permission by checking permission on each tRPC endpoints, wrapped each handler into createProtectedHandler, and show a message to user without permission.
```

Comment:
This is not a changelog, this is a commit message. Please write a little about the entire change context, including fact that Dashboard will now enable apps read for everyone